docs/performance/storage.md

```diff
@@ -19,26 +19,34 @@ We are going to use [Yellow taxi trip - January 2024](https://www.nyc.gov/site/t
 | 2        | 1704074134000        | 1704074399000         | 1               | 1.08          | 1          | "N"                | 249          | 68           | 1            | 7.2         | 1     | 0.5     | 2.44       | 0            | 1                     | 14.64        | 2.5                  | 0           |
 ```
 
-Will take:
-
+The taxi dataset takes:
 - ~48MiB in Parquet (very optimized for storage)
 - ~342MiB in CSV
 - ~1.2GiB in JSON
 - ~510MiB in PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1)
 - ~775MiB in CrateDB 5.9.3 (3 nodes, default settings)
+- ~431MiB in CrateDB 5.10.2 (3 nodes, default settings)
+
+We will dive deeper to really understand what is going on.
 
-At first sight, it might look that CrateDB storage takes more than PostgreSQL,
-but we need to dive deeper to really understand what is going on.
+> INFO: In version 5.10 storage usage was improved, some users report up to 70% of storage reduction,
+> more reasonably a 30–50% can be expected for most data, the reduction % of reduction will vary
+> depending on use cases. 
```

Comment:
FYI: In MyST Markdown, you may want to phrase the [admonition](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html) like this:
```suggestion
:::{note}
In version 5.10 storage usage was improved, some users report up to 70% of storage reduction,
more reasonably a 30–50% can be expected for most data, the reduction % of reduction will vary
depending on use cases. 
:::
```